progress/render.go

```diff
@@ -266,16 +272,16 @@ func (p *Progress) renderTrackerProgress(out *strings.Builder, t *Tracker, track
 		}
 		p.renderTrackerStats(out, t, hint)
 		out.WriteString(p.style.Colors.Message.Sprint(p.style.Options.Separator))
-		p.renderTrackerMessage(out, t)
+		p.renderTrackerMessage(out, t, message)
 		out.WriteRune('\n')
 	}
 }
 
-func (p *Progress) renderTrackerMessage(out *strings.Builder, t *Tracker) {
+func (p *Progress) renderTrackerMessage(out *strings.Builder, t *Tracker, message string) {
```

Comment:
The goal with this change is to make sure that if the renderer edits the message, that the edited message is the message that is being displayed.

While not a formal race condition, without this change if UpdateMessage was called by the user between the text.Pad/text.Snip and this point the displayed message + progressbar might (for a very short period of time) not look as expected.